js/main.js

```diff
@@ -66,12 +67,37 @@ function toggleCollection(event) {
     toggle === "true" ? "Hide Full Collection" : "Show Full Collection";
 }
 
+function updateGachaView(e){
+  gacha_display_selection = e.target.value;
+  if(gacha_display_selection == "gacha-grid"){
+    // make this a grid
+    GACHA_SECTION.classList.add("grid-display");
+    GACHA_SECTION.classList.remove("pile-display");
+    addHover();
+  } else {
+    // make this a pile
+    GACHA_SECTION.classList.remove("grid-display");
+    GACHA_SECTION.classList.add("pile-display");
+    removeHover();
+  }
+}
```

Comment:
(nit) this function be moved to cards.js since it's importing a bunch of things from it anyways. Then only one import is needed.
I don't feel strongly about this, feel free to keep as-is

Reply:
I have updated this and moved it to cards.js.